app/js/service/rate.js

```diff
@@ -12,7 +12,7 @@
   const USE_CORS_PROXY = RSBP_CONFIG.rate.useCorsProxy;
   const EXPIRATION = RSBP_CONFIG.rate.expiration;
   const IS_BTC = CURRENCY === "BTC";
-  const URL = "https://api.bitcoinaverage.com/ticker/" + CURRENCY + "/last";
+  const URL = "https://apiv2.bitcoinaverage.com/indices/global/ticker/" + CURRENCY + "/last";
```

Comment:
It appears this line is not returning the price as expected. 

![screen shot 2017-08-16 at 2 37 30 pm](https://user-images.githubusercontent.com/4895224/29384206-820a0306-8290-11e7-8975-f2b5b9b55ba5.png)